demos/android/MASVS-CRYPTO/MASTG-DEMO-0015/run.sh

```diff
@@ -0,0 +1 @@
+ semgrep -c ../rules/mastg-android-hardcoded-crypto-keys-usage.yml ./hardcoded-key-in-use.java --text -o output.txt  
```

Comment:
This must work on MastgTest_reversed.java

```suggestion
 semgrep -c ../rules/mastg-android-hardcoded-crypto-keys-usage.yml ./MastgTest_reversed.java --text -o output.txt  
```